core/common/src/main/java/alluxio/master/ZkMasterInquireClient.java

```diff
@@ -78,16 +97,20 @@ public static synchronized ZkMasterInquireClient getClient(String zookeeperAddre
    *
    * @param connectDetails connect details
    * @param electionPath the path of the master election
+   * @param inquireRetryCount the number of times to retry connections
+   * @param authType Alluxio auth type
    */
   private ZkMasterInquireClient(ZkMasterConnectDetails connectDetails, String electionPath,
-      int inquireRetryCount) {
+      int inquireRetryCount, AuthType authType) {
     mConnectDetails = connectDetails;
     mElectionPath = electionPath;
 
     LOG.info("Creating new zookeeper client for {}", connectDetails);
-    // Start the client lazily.
-    mClient = CuratorFrameworkFactory.newClient(connectDetails.getZkAddress(),
-        new ExponentialBackoffRetry(Constants.SECOND_MS, 3));
+    CuratorFrameworkFactory.Builder curatorBuilder = CuratorFrameworkFactory.builder();
+    curatorBuilder.connectString(connectDetails.getZkAddress());
+    curatorBuilder.retryPolicy(new ExponentialBackoffRetry(Constants.SECOND_MS, 3));
+    curatorBuilder.zookeeperFactory(new NoSaslZookeeperFactory());
```

Comment:
i don't get it. where is authType used now?

Reply:
It's for future use.